background/services/indexing/index.ts

```diff
@@ -590,13 +590,17 @@ export default class IndexingService extends BaseService<Events> {
     asset: SmartContractFungibleAsset,
     addressNetwork: AddressOnNetwork
   ): Promise<void> {
-    const { metadata = {} } = asset
-    // Manually imported tokens are verified
-    metadata.verified = true
+    // eslint-disable-next-line no-param-reassign
+    asset.metadata = {
+      ...(asset.metadata ?? {}),
+      // Manually imported tokens are verified
+      verified: true,
+    }
```

Comment:
Can we avoid that reassign?
```suggestion
    const customAsset = {
      ...asset,
      metadata: {
         ...(asset.metadata ?? {}),
         // Manually imported tokens are verified
         verified: true,  
      }
    }
```